test/ui-test/initializationUITest.ts

```diff
@@ -29,17 +29,28 @@ export function knativeInitializationUITest(): void {
       } catch (error) {
         assert.fail(`Error notification appeared during cluster loading`);
       }
-      // eslint-disable-next-line @typescript-eslint/no-floating-promises
-      driver.wait(async () => !(await sideBar.getContent().hasProgress()), 3000);
-      expect(await sideBar.getContent().getText()).to.equal(KNativeConstants.NO_SERVICE_FOUND);
-      const sections = await sideBar.getContent().getSections();
-      expect(sections.length).to.equal(1);
-      const section = sections[0];
-      expect(await section.getTitle()).to.equal(KNativeConstants.KNATIVE_EXTENSION_NAME);
-      const items = await sections[0].getVisibleItems();
-      expect(items.length).to.equal(1);
-      const item = items[0];
-      expect(await item.getText()).to.equal(KNativeConstants.NO_SERVICE_FOUND);
+      await driver.wait(async () => !(await sideBar.getContent().hasProgress()), 3000);
```

Comment:
Is the wait supposed to be 3 thousand or 30 thousand? The ones below are 30.

Reply:
3 thousands miliseconds it is. there is inversion of the condition, so I am basically waiting for something to start.

Reply:
No, you are right, this seems to not be exactly doing what I would expect it to do. I will revise it.